FLT/Mathlib/Topology/Algebra/RestrictedProduct.lean

```diff
@@ -205,17 +207,17 @@ def Homeomorph.restrictedProductProd {ι : Type*}
       continuous_invFun := sorry -- FLT#568
 
 -- Is there a mathlibism for {f | ∀ j, f j ∈ C j i}?
+-- Yes: Set.pi Set.univ
```

Comment:
So can you change `Equiv.restrictedProductPi` to use this, or is there a catch?

Reply:
The catch is defeqs: Set.pi Set.univ would give `{f | ∀ j ∈ univ, f j ∈ C j i}` which `simp`s to what you want, but isn't exactly the same. imo we should keep this as is since it's easier to use, but it doesn't make a big difference either way